src/components/Dashboard/Search.vue

```diff
@@ -1,142 +1,353 @@
+                          <!-- if cash show number else - -->
+                          {{
+                            item.paymentAccount &&
+                            item.paymentAccount.paymentMethod === 'CASH'
+                              ? item.payments &&
+                                item.payments[0] &&
+                                item.payments[0].chequeReceiptNumber
+                              : '-'
+                          }}
+                        </td>
+                        <td v-if="canShowColumn('date')">
+                          {{ item.routingSlipDate }}
+                        </td>
+                        <td v-if="canShowColumn('status')">
+                          <span
+                            :class="colors(item.status)"
+                            class="font-weight-bold"
+                            data-test="label-status"
+                            >{{ getStatusLabel(item.status) }}</span
+                          >
+                        </td>
+                        <td v-if="canShowColumn('folioNumber')">
+                          <span
+                            v-if="
+                              formatFolioResult(item).length > 0 &&
+                                !showExpandedFolio.includes(item.id)
+                            "
+                            @click="toggleFolio(item.id)"
+                            class="cursor-pointer"
+                          >
+                            {{ formatFolioResult(item)[0] }}
+                            <v-icon
+                              small
+                              v-if="formatFolioResult(item).length > 1"
+                              color="primary"
+                            >
+                              mdi-menu-down</v-icon
+                            ></span
+                          >
+                          <template v-if="showExpandedFolio.includes(item.id)">
+                            <div
+                              v-for="(folio, index) in formatFolioResult(item)"
+                              :key="index"
+                              @click="index === 0 ? toggleFolio(item.id) : ''"
+                              :class="index === 0 ? 'cursor-pointer' : ''"
+                            >
+                              <span>
+                                {{ folio }}
+                                <v-icon
+                                  small
+                                  v-if="index === 0"
+                                  color="primary"
+                                >
+                                  mdi-menu-up</v-icon
+                                ></span
+                              >
+                            </div>
+                          </template>
+                        </td>
+                        <td v-if="canShowColumn('initiator')">
+                          {{
```

Comment:
Initiator is not mandatory, so you would need to show '-' if there are no initiator.